src/Ampersand/PatchHelper/Command/AnalyseCommand.php

```diff
@@ -56,7 +58,11 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
                 $output->writeln("<info>Validating $file</info>", OutputInterface::VERBOSITY_VERBOSE);
 
-                foreach ($patchOverrideValidator->validate()->getErrors() as $errorType => $errors) {
+                $vendorNamespaces = [];
+                if ($input->getOption('vendor-namespaces')) {
+                    $vendorNamespaces = explode(',', $input->getOption('vendor-namespaces'));
```

Comment:
I recommend trimming the namespaces, so that `"Namespace1, Namespace2"` works :)

Reply:
Good point, thanks. Did that now.